app/controllers/tmdb_controller.rb

```diff
@@ -30,7 +30,7 @@ def person_autocomplete
 
   def movie_more
     if params[:tmdb_id]
-      @movie = Tmdb::Client.movie(params[:tmdb_id])
+      @movie = GuaranteedMovie.find_or_initialize_from_api(params[:tmdb_id])
```

Comment:
Ah, okay this is interesting. So here we're choosing to check the db first instead of going straight to the api.

Reply:
Yeah, seems like we _should_ know at this point it's not in the db, but since we were doing the db check before (because it was inside `MovieMore`), and since I was too lazy to searching for everywhere calling this method, I kept it that way.